client/components/dialog/dialog-base.jsx

```diff
@@ -80,7 +77,7 @@ class DialogBase extends Component {
 		}
 
 		return (
-			<div className={ buttonsClassName } ref="actionButtons">
```

Comment:
Have we searched to see if there any usages of the actionsButtons or content ref?

Reply:
Yup. eslint was complaining about it (string literals are deprecated) so I removed them after confirming they are not used.